compiled_starters/cpp/src/main.cpp

```diff
@@ -7,7 +7,7 @@ int main() {
 
   // Uncomment this block to pass the first stage
   // std::cout << "$ ";
-  //
-  // std::string input;
-  // std::getline(std::cin, input);
+  
+  std::string input;
+  std::getline(std::cin, input);
```

Comment:
@andy1li this is a good idea, but just want to confirm - will this cause the tester to end up waiting 10 seconds (test timeout) for the program to terminate? Or will we terminate early?

Reply:
Don't know yet, but I can test and report back.

For context, all starter code expect cpp are already looking like this. C++ is the odd one out.

Reply:
@rohitpaulk  Result: We terminate early when "$ " is not found:

<img width="1004" alt="image" src="https://github.com/user-attachments/assets/58babf4b-12a0-4452-be35-4a18bbe9ff7b" />

Reply:
Gotcha, looks good!

Reply:
@rohitpaulk Does the () look weird to you as well? I don't remember if it's there before.

<img width="748" alt="image" src="https://github.com/user-attachments/assets/38eadf08-557a-4c4a-afdd-983132d66127" />